EasyReflectometryApp/Logic/Proxies/Project.py

```diff
@@ -231,17 +238,24 @@ def _loadProject(self):
                     self.parent._interface.switch(inter)
 
         self.parent._model_proxy._colors = descr['colors']
+
         self.parent._model_proxy._model = ModelCollection.from_dict(descr['model'])
```

Comment:
similarly all other assignments to other proxies. Those should be done with signals/slots

Reply:
Cool